### ISSUE TITLE
Develop storage policy

### ISSUE THREAD

**Issue by petebachant** [UNH-OE/chase-lab#3]
To avoid the giant annual cleanup effort, a storage policy should be developed and enforced. The goal is to reach a steady state, where stuff coming into the lab must be balanced by stuff moving out. 
## Some example policies
1. **None** (current): Find an empty spot for your stuff and keep it there until someone complains.
2. **Tag-your-stuff**: Put a tag with contact information on your stuff so someone knows not to take your stuff and who to complain to if it's in their way.
3. **Lease-your-space**:
   - Floor and shelf spaces are marked off and identified/indexed ([example spreadsheet](https://docs.google.com/spreadsheets/d/1Ozrztwq1ItJVnXmXyqE_aEpuYlVGxJ1J4ShSshFW9zM/edit?usp=sharing)).
   - Students and faculty "lease" space for their project for a finite time, after which the space must be emptied, which could mean disposal, or moved to a "public supply area," which also has a finite capacity.
     - Faculty could have some "owned" space as well.
   - There are "lost and found" and public supply areas that when filled, are limited by disposing or scrapping materials within.

**Comment by petebachant** [comment]
This policy could be written at https://github.com/UNH-OE/chase-lab/wiki/Storage

**Comment by wosnik** [comment]
I think under the current model we are moving to the 3rd option.

**Comment by wosnik** [comment]
Hey Pete

I think we can solve most of the issues with me just being involved and building relationships with people. And by having a director that's interested in maintaining order in the space. Relationships are more likely to build a sense of community and common regard. Which would lead to people having more respect for each other and the space.

The space is a mess because it had no real oversight.

**Comment by petebachant** [comment]
I agree that will help, but I think you have too much faith in us students! 

I also think people will respect the space when they are accountable for a clearly-defined chunk of it. For example, imagine the floor is taped off in a grid, and you notice there is junk overflowing out of spot A5 or something. Anyone could look up in the database and see who to get in touch with. Or, if their "lease" is up, it would be easy to know if it's okay to clear it out. 

Another part of the policy could be that anything left out in a public space, e.g., a workbench, is free game. This would 
1. Incentivize people to not leave stuff in shared spaces
2. Enable others to _mercilessly_ clean shared spaces if they felt so inclined

**Comment by jcahern** [comment]
i have faith in myself!

I think we need both.  I'm currently working on the relationship side 
and determining how the space is used and how it should be divided.  but 
most of that will prob be decided without me.

the space can't be operated on autopilot under those set of rules. human 
behavior just doesn't conform.  interaction is based on relationships.  
an ethic of community will do a lot to solve the same problem.  for 
example what if someone is in a hurry and pulls in and drops off some 
stuff and then goes to park in b lot but by the time they get back 
someone got to'd that there's stuff in their space so they threw it in 
the dumpster.  strict conformance to rules by itself will not solve the 
problem.  it will make people more territorial and devisive.  but if 
everyone feels like they are being treated fairly and their space is 
respected it should result in better collegiality.  we need a space 
policy but also cultural change.

**Comment by petebachant** [comment]
I have faith in you too John!

**Comment by wosnik** [comment]
I think we need a well-defined general structure, i.e., what goes where,  and defined usage spaces, i.e., workbenches and shelf space given to certain senior projects, storage space given to PIs etc.
I’m for marking certain floor areas that must be kept clear at all times, but other than that “project space” is a bit of a moving target (example: the ROV room that had to be built a couple years ago, and everything around it had to move…)

Generally speaking I would like to operate under a culture of “continual improvement”:
- ```
       leave it better than you found it.
  ```
- ```
       If you can spend some time to improve a process for everybody, consider doing that.
  ```
  
  If you look at my group, we have invested significant time and money in this process, e.g. Pete’s renovation of the tow mechanism, Ivo’s renovation of the cavitation tunnel. (Albeit, I am also guilty of having lots of stuff all over the lab.)
  It will take some time to change to a culture of “continual improvement”, and I think it will go a long way if we all start to just pay attention and treat the lab space with some respect, and say something to people who don’t. A lot of the undergraduates who end up leaving a mess simply mimicked the behavior they observed from everybody else, so let’s change that.

Martin

**Comment by petebachant** [comment]
w.r.t flexibility my thoughts were that you could allocate multiple adjacent storage space "units" for projects as necessary. People could work on their stuff in a shared area, but if they're not present, everything must be put away in their "owned" space. In other words, don't leave your stuff on a shared workbench when you leave for the day. This seems like a rigid policy but I think it's the simplest option. Otherwise the management will spend a lot of effort dealing with corner cases and exceptions, tracking people down to find out whose stuff is what, and probably moving stuff that might be important to someone. 

Anyway, does anyone want to take a stab at drafting up a storage policy at https://github.com/UNH-OE/chase-lab/wiki/Storage?
